content/en/tracing/profiling/_index.md

```diff
@@ -0,0 +1,91 @@
+| Runtime  | The type of runtime the profiled process is running (`JVM`, `CPython`). |
+| Language | The language of your code (`Java`, `Python`).                           |
+
+The following measures are available:
+
+|  Measure          | Definition                                                                                                                                                                          |
+| ----------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU               | CPU load summed on all cores of the process. This means this can go above `100%` and the theoretical limit is `nCores` \* `100%`.                                                    |
+| Memory Allocation | Memory allocation rate in `Bytes/s` over the course of the profile. This value can be above the maximum amount of RAM memory because it can be garbage collected during the profile. |
+
+## Profile
+
+Click on one line to view a specific profile:
+
+{{< img src="tracing/profiling/profile.png" alt="A specic profile">}}
+
+The profile header contains information associated with your profile, like: the service that generated it, or the environment and code version associated to it. In the upper right corner of each profile, there are options to:
+
+- Download the profile
+- Switch the profile to full screen
+
+Four tabs are below the profile header:
+
+| Tab          |  Definition                                                                                                                                 |
+| ------------ | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| Profiles     | Flame graph and summary table of the profile you are looking at. You can switch between multiple profile types (`CPU`, `Memory allocation`) |
+| Analysis     | A set of heuristics that suggest potential issues or areas of improvement in your code                                                      |
+| Metrics      | Profiling metrics coming from all profiles of the same service                                                                              |
+| Runtime Info | Text dump of all the Runtime properties                                                                                                     |
+
+### Profile types
+
+In the **Profiles** tab you can see all profile types available for a given language. Depending on the language, the information collected about your profile differs. Refer to the dedicated profile types section for your language to see what is collected:
+
+{{< partial name="profiling/profiling-languages.html" >}}
+
+## Troubleshooting
+
+Datadog profiling is in beta. Note the following limitations:
+
+- Datadog profiling does not work with serverless.
```

Comment:
```suggestion
- Datadog profiling does not yet work with serverless.
```

Reply:
We avoid wording like "currently, yet" in the doc. The doc is timeless.